certora/specs/SocialRecoveryModule.spec

```diff
@@ -1,3 +1,27 @@
-rule demo() {
-    assert true;
+using SafeHarness as safeContract;
+
+methods {
+    // Safe Functions
+    function safeContract.isModuleEnabled(address) external returns (bool) envfree;
+    function safeContract.disableModule(address, address) external;
+}
+
+// A setup function that requires Safe contract to enabled the Social Recovery
+// Module.
+function setupRequireSafeTokenInvariants() {
+    require(safeContract.isModuleEnabled(currentContract));
+}
+
+// This is a dummy rule to verify the Safe Contract Setup with the Social Recovery
+// Module is working as intended.
+rule SocialRecoveryModuleEnabled {
```

Comment:
The rule name does not correspond to the assertion.
For me `recoveryModuleCanBeDisabled` seems an appropriate name if thats the purpose of the rule.

Reply:
Done 👍🏾